Formula/orientdb.rb

```diff
@@ -39,4 +39,14 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    begin
+      system "#{bin}/orientdb", "start"
```

Comment:
I just figured out that there's a `server.sh` file that I can run instead of the daemon, and the logging errors are now getting output correctly to the console on test. `/usr/local/Cellar/orientdb/2.2.0/libexec/bin/server.sh: line 95: /usr/local/Cellar/orientdb/2.2.0/libexec/bin/orient.pid: Operation not permitted
Can't load log handler "java.util.logging.FileHandler"...` Hopefully solution to follow shortly.